.github/workflows/jsdoc.yml

```diff
@@ -0,0 +1,47 @@
+name: JSDoc
+on:
+  pull_request:
+    paths:
+      # documentation
+      - "docs/**"
+      - "*.md"
+      # changes to this workflow
+      # seems there is no way to get this easily yet https://github.com/actions/runner/issues/853
+      - ".github/workflows/jsdoc.yml"
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.0
+
+env:
+  REALM_DISABLE_ANALYTICS: 1
+jobs:
+  jsdoc:
+    name: JSDoc
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+        with:
+          submodules: "recursive"
+      - uses: actions/setup-node@v1
+        with:
+          node-version: 14
+          registry-url: https://registry.npmjs.org/
+      - name: Install npm v7
+        run: npm install -g npm@7
+      # Install the root package (--ignore-scripts to avoid downloading or building the native module)
+      - name: Install root package dependencies
+        run: npm ci --ignore-scripts
+      - name: Install subpackages dependencies
+        run: npm run lerna:bootstrap
```

Comment:
Are you sure this is needed?

```suggestion
```

Reply:
I had copied this from another workflow, I will remove it.